worlds/marioland2/locations.py

```diff
@@ -0,0 +1,61 @@
+locations = {
+    'Mushroom Zone': {'id': 0x00, 'ram_index': 0, 'type': 'level'},
```

Comment:
Our style guide says to use double quotes `"` for strings

Reply:
Much of this was output using Python Console to process data, this is how it is output.

Reply:
> Our style guide says to use double quotes `"` for strings

Changed